Wpb 26240 corrections (#123)
wireapp/wire-docs: src/understand/federation/glossary.rst

Asset

   Any file or image sent via Wire (uploaded to and downloaded from a backend).

.. _qualified-user-id:

Qualified User Identifier (QUID)

  A combination of a UUID (unique on the user's backend) and a domain.

.. _qualified-user-name:

> Qualified User Identifier (QUN)

Reviewer: ```suggestion
Qualified User Name (QUN)
```

  A combination of a name that is unique on the user's backend and a domain. The
  name is a string consisting of 2-256 characters which are either lower case
  alphanumeric, dashes, underscores or dots. See `here
  <https://github.com/wireapp/wire-server/blob/f683299a03207acb505254ff3121213383d0b672/libs/types-common/src/Data/Handle.hs#L76-L93>`_

>   for the code defining the rules for user names.

Reviewer: ```suggestion
  for the code defining the rules for user names. Note that in the wire-server source code, user names are called 'Handle' and qualified user names 'Qualified Handle'.
```

.. _qualified-device-id:

Qualified Device Identifier (QDID)

>   A combination of a randomly generated string of either 10 or 15 characters

Reviewer: ```suggestion
  A combination of a device identifier (a hash of the public key generated for a user's device)
```

  concatenated with a dot and the QUID of the associated user. The resulting

>   string is unique on the backend of the associated user.

Reviewer: I suppose it's even globally unique, not just on that backend, as the QUID contains a domain, right?
